straxen/contexts.py

```diff
@@ -272,6 +272,7 @@ def xenonnt_simulation(output_folder='./strax_data'):
     electron_drift_velocity=("electron_drift_velocity_constant", 1.3325e-4, False),
     event_info_function='disabled',
     max_drift_length=96.9,
+    electron_drift_time_gate=("electron_drift_time_gate_constant", 1700),
```

Comment:
@jingqiangye do you happen to know the correct value for 1T?

Reply:
Yeah I think it is the `electron_drift_time_gate_constant` for 1T, see [Tab. 1](https://xe1t-wiki.lngs.infn.it/doku.php?id=dandrea:diffusionstudywithkrdata)